src/main/scala/mastermind/Mastermind.scala

```diff
@@ -12,20 +12,38 @@ type Board = String
 val validColors = List('B', 'Y', 'R', 'G')
 
 /** Get a random color from the list of valid colors */
-def getRandomColor(): Color =
-  ???
+def getRandomColor(): Color = {
+  val rand = scala.util.Random()
+  val colorIndex = rand.nextInt(4)
+  validColors(colorIndex)
+}
 
 /** Given four colors, make a board from them */
 def makeBoardFromColors(c1: Color, c2: Color, c3: Color, c4: Color): Board =
-  ???
+  val colorSeq = List(c1, c2, c3, c4)
+  colorSeq.mkString
 
 /** Create a random board */
 def getRandomBoard(): Board =
-  ???
+  val c1 = getRandomColor()
+  val c2 = getRandomColor()
+  val c3 = getRandomColor()
+  val c4 = getRandomColor()
+
+  makeBoardFromColors(c1, c2, c3, c4)
```

Comment:
I also did this, but I felt a bit weird about assigning variables so repetitively like this. I wish there was some way to unpack a list into arguments like how python has the asterisk.